src/App/Handler/MaintenanceOverviewHandler.php

```diff
@@ -24,12 +24,9 @@ class MaintenanceOverviewHandler implements RequestHandlerInterface
         'laminas-mvc-plugin-prg',
         'laminas-mvc-middleware',
         'laminas-mvc-i18n',
-        'laminas-mvc-psr7bridge',
+        'laminas-psr7bridge',
```

Comment:
Is not a MVC component.

Reply:
The [laminas-developer-tools](https://github.com/laminas/laminas-developer-tools) are missing.

Reply:
added

Reply:
https://preview-lf246gy-2ja7ciew2nbkm.us-2.platformsh.site/packages-maintenance-status/